doc/RecordingControl.xml

```diff
@@ -1684,8 +1787,420 @@ SetRecordingJobMode(JobToken, Active)
+        If a <literal>Postfix</literal> is configured for the recording, the device shall include <literal>postfix</literal> in <literal>lead</literal> and replace it with the configured <literal>Postfix</literal>.
+        The device shall replace <literal>date</literal> and <literal>time</literal> respectively with the date and time portion of the first sample contained in the segment in Universal Coordinated Time (UTC), following the ABNF rules:
+        <itemizedlist>
+          <listitem>
+            <para><literal>year</literal> represents the 4-digit year</para>
+          </listitem>
+          <listitem>
+            <para><literal>month</literal> represents the 2-digit month (01 to 12)</para>
+          </listitem>
+          <listitem>
+            <para><literal>day</literal> represents the 2-digit day of the month (01 to 31)</para>
+          </listitem>
+          <listitem>
+            <para><literal>hour</literal> represents the 2-digit hour (00 to 23)</para>
+          </listitem>
+          <listitem>
+            <para><literal>minute</literal> represents the 2-digit minute (00 to 59)</para>
+          </listitem>
+          <listitem>
+            <para><literal>second</literal> represents the 2-digit second (00 to 60, due to leap second rules)</para>
+          </listitem>
+          <listitem>
+            <para><literal>secfrac</literal> represents the optional fractional seconds with a precision of up to 6 digits</para>
+          </listitem>
+        </itemizedlist>
+        The device shall replace <literal>counter</literal> with a span-dependant counter set to 0 for a new span and incremented by 1 when closing a segment of this span.
+        The device shall replace <literal>extension</literal> with the extension defined by the type of span the segment belongs to.
+      </para>
+      <para>
+        Note: <literal>date</literal> is conformant to the extended date format defined in ISO 8601-1.
+        <literal>time</literal> is <emphasis role="bold">not</emphasis> conformant to the extended time format defined in ISO 8601-1.
+      </para>
+    </section>
+    <section xml:id="_refCloudRecordingSpanEndObjects">
+      <title>Span end objects</title>
+      <para>
+        To store the end time of a span, the device should store an additional empty object when closing a span.
+        The device shall generate the object key for this object according to the ABNF <literal>key</literal> rule and replacement rules defined in <xref linkend="_refCloudRecordingSegmentObjects" />.
+        The device shall replace <literal>date</literal> and <literal>time</literal> respectively with the date and time portion of the end of the last sample contained in the span in Universal Coordinated Time (UTC), following the ABNF rules.
```

Comment:
given an example for the object key when the span ends.

Reply:
thanks @fschuetz04 for adding example section.

> > please provide an example in the spec document for the prefix of the attributes based on storage format configured for the recording.
> 
> @venki5685 I added examples for the attribute prefixes.

@fschuetz04 thanks for adding example section for attribute prefixes.